Getting-Started.md

```diff
@@ -166,61 +149,62 @@ upstream	https://github.com/chaoss/grimoirelab-perceval (push)
 
 In order to add a remote to a Git repository, you can use the following command:
 ```
-~/sources/perceval$ git remote add upstream https://github.com/chaoss/grimoirelab-perceval
+$ git remote add upstream https://github.com/chaoss/grimoirelab-perceval
 ```
 
-Then, you can download the [PyCharm community edition](https://www.jetbrains.com/pycharm/download/#section=linux), using any of the following alternatives:
-
-* **Alternative 1: Install PyCharm via Snap Package Management**
-    
-    Install Snap Package Management using the below command.
-
-     `sudo apt install snapd snapd-xdg-open`
-    
-    Now that `Snap` is installed, run the command below to install the community version of PyCharm.
-    
-     `sudo snap install pycharm-community --classic`
-
-* **Alternative 2: Install PyCharm via Ubuntu Software Center**
-
-    Open Ubuntu Software Center and search for PyCharm. Then, select and install the community version of PyCharm.
-
-* **Alternative 3: Download PyCharm tar file**
+#### ProTip [&uarr;](#cloning-the-repositories-)
 
-    Go to the official website of [JetBrains]((https://www.jetbrains.com/pycharm/download/#section=linux)) and download the community version of PyCharm for your system.
-    Then, extract the downloaded `tarfile` file using the below command.
-    
-    `tar -xzf tarfile`
+You can use this use this [script]() to automate this whole process.
+```
+$ python3 glab-dev-setup.py --create --token xxxx --source sources
+```
 
-    Go to the `bin` directory and run the PyCharm shell.
+### Setting up PyCharm [&uarr;](#source-code-and-docker-)
 
-After the installation is complete, this [tutorial](https://www.jetbrains.com/help/pycharm/quick-start-guide.html)
-can be followed to gain familiarity with PyCharm.
+> Help: 
+> You need to install PyCharm (**Community Edition**) for this. Please refer the documentation.
+> - https://www.jetbrains.com/help/pycharm/installation-guide.html
+>
+> You can follow this [tutorial](https://www.jetbrains.com/help/pycharm/quick-start-guide.html) to get familiar with PyCharm.
```

Comment:
I removed the installation procedure for PyCharm, this was in reference to this discussion https://github.com/chaoss/grimoirelab-sirmordred/pull/447#discussion_r399674801.

Please let me know if it is good to add it back.